packages/fxa-customs-server/lib/settings/requestChecks.js

```diff
@@ -21,9 +21,17 @@ module.exports = (config, Settings, log) => {
       if (!this.flowIdExemptUserAgentREs) {
         this.flowIdExemptUserAgentCompiledREs = [];
       } else {
-        this.flowIdExemptUserAgentCompiledREs = this.flowIdExemptUserAgentREs.map(function(re) {
-          return new RegExp(re);
-        });
+        this.flowIdExemptUserAgentCompiledREs = this.flowIdExemptUserAgentREs.map(
+          function(re) {
+            // Log the regex at startup to check that we have correct values.
+            const regex = new RegExp(re);
+            log.info({
+              op: 'requestChecks.setAll.regex',
+              regex: String(regex),
```

Comment:
This string coercion is to compensate for the logging code in the module being dumb.